### PR TITLE
Esxcli command version retrieval

### PR DESCRIPTION
**Pull request by uppalk1**
jenkins: depends on RackHD/on-tasks#446

### CODE REVIEW

**Comment by uppalk1** [comment]
@RackHD/corecommitters @geoff-reid @tannoa2 @keedya

File: lib/graphs/esxcli-driver-version-commands-graph.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+//Copyright © 2017 Dell Inc. or its subsidiaries. All Rights Reserved.
+
+'use strict';
+
+module.exports = {
+    friendlyName: 'ESXi Driver Version Retrieval',
+    injectableName: 'Graph.Driver.Version.Retrieval',
+    options: {
+        defaults: {
+            commands: null
```
Comment (anhou) [inline lib/graphs/esxcli-driver-version-commands-graph.js]: not esxcli command here?

File: lib/graphs/lsi-mr3-ssh-commands-graph.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+//Copyright © 2017 Dell Inc. or its subsidiaries. All Rights Reserved.
+
+'use strict';
+
+module.exports = {
+    friendlyName: 'ESXi Driver Version Retrieval',
+    injectableName: 'Graph.Driver.Version.Retrieval',
```
Comment (anhou) [inline lib/graphs/lsi-mr3-ssh-commands-graph.js]: what's this file 'lsi-mr3-*' used?  and injectable name is the same with above, a mistake?

File: lib/graphs/esxcli-driver-version-commands-graph.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+//Copyright © 2017 Dell Inc. or its subsidiaries. All Rights Reserved.
+
+'use strict';
+
+module.exports = {
+    friendlyName: 'ESXi Driver Version Retrieval',
+    injectableName: 'Graph.Driver.Version.Retrieval',
```
Comment (anhou) [inline lib/graphs/esxcli-driver-version-commands-graph.js]: the injectableName  include 'ESXi' is more accurate

File: lib/graphs/esxcli-driver-version-commands-graph.js (hunk 1 above)
Comment (uppalk1) [inline lib/graphs/esxcli-driver-version-commands-graph.js]: We pass it through the payload.

File: lib/graphs/lsi-mr3-ssh-commands-graph.js (hunk 2 above)
Comment (uppalk1) [inline lib/graphs/lsi-mr3-ssh-commands-graph.js]: I'll remove this file

**Comment by geoff-reid** [comment]
test this please

**Pull request merged**